languages.json

```diff
@@ -34,8 +34,10 @@
     "is": "Íslenska",
     "it": "Italiano",
     "kn": "ಭಾಷೆ-ಹೆಸರು",
+    "kk": "Қазақша",
     "rw": "Kinyarwanda",
     "ht": "Kreyòl",
+    "kk": "Қазақша",
```

Comment:
Looks like you forgot to remove the original key - that's why the build is failing